verticaltabs.js

```diff
@@ -794,6 +795,9 @@ VerticalTabs.prototype = {
 
   clearFind: function (purpose) {
     if (this.document.getElementById('find-input')){
```

Comment:
Since we're using this so often, we should just make a variable out of it.
``` javascript
let findInput = this.document.getElementById('find-input');
if (findInput) {
  if (findInput.value === '') {
...
```